### ISSUE TITLE
Organise color palette

### ISSUE THREAD

**Issue by omahs** [volkyeth/noundry-studio#23]
<img width="180" alt="image" src="https://user-images.githubusercontent.com/73983677/214563084-19f03542-5a23-4eee-bea3-83b052be68e2.png">
It's a great Noun editor! I was just surprised by the order of the colors, I suggest to organise them like a rainbow.

**Comment by volkyeth** [comment]
@omahs thanks for the feedback!

Turns out, arranging an uneven color palette in a neat way is a much harder problem than it seems 😅 
ref: https://www.alanzucconi.com/2015/09/30/colour-sorting/

I've tried my hand at it, but since it's pretty uneven, it ended up looking worse when algorithmically sorted. It probably emerged organically from the first set of Noun traits, so that's why the colors are not equally spaced.

Since the main purpose of Noundry Studio is to create art that we can upload to the Nouns contract later, a good solution is #20: proposing a new, symmetric palette to the Nouns contract, and adding that one to Noundry studio.

I do intend to add support for multiple palettes, but I could also add a loose palette mode with free color picking instead of a fixed palette.

What's your use case? Are you doing art for Nouns/derivative contracts?
